test/controllers/income/estimatedPay/update/IncomeUpdateHowToUpdateControllerSpec.scala

```diff
@@ -68,10 +68,10 @@ class IncomeUpdateHowToUpdateControllerSpec
         mock[FormPartialRetriever],
         MockTemplateRenderer
       ) {
-    when(journeyCacheService.mandatoryValueAsInt(Matchers.eq(UpdateIncome_IdKey))(any()))
-      .thenReturn(Future.successful(employer.id))
-    when(journeyCacheService.mandatoryValue(Matchers.eq(UpdateIncome_NameKey))(any()))
-      .thenReturn(Future.successful(employer.name))
+    when(journeyCacheService.mandatoryJourneyValueAsInt(Matchers.eq(UpdateIncome_IdKey))(any()))
+      .thenReturn(Future.successful(Right(employer.id)))
+    when(journeyCacheService.mandatoryJourneyValue(Matchers.eq(UpdateIncome_NameKey))(any()))
+      .thenReturn(Future.successful(Right(employer.name)))
   }
```

Comment:
Missing left tests

Reply:
done

Reply:
Will do as discussed. This is the very first page. Will write a test for the left case here which will only test the handleChooseHowToUpdate (used on Post) as howToUpdatePage(Used on Get) will not use the create function which return the Either. For all other pgaes I have written the test for the controller used on the GET and not the post. I will write tests for all the controllers used on the POST now.

Reply:
done